python/src/FunctionalChaosAlgorithm_doc.i.in

```diff
@@ -46,94 +46,66 @@ weights : sequence of float
 
 Notes
 -----
-Consider a physical model :math:`g` depending
-on a set random variables :math:`\vect{X}=(X_1, \dots, X_{n_X})^T`.
-We call functional chaos expansion the class of spectral methods which gathers
-all types of response surface that can be seen as a projection of the physical
-model in an orthonormal basis. This class of methods uses the Hilbertian space
-(square-integrable space: :math:`L^2`) to construct the response surface.
 
-The functional chaos representation implemented in OpenTURNS is the
-polynomial chaos (PC) expansion. It allows one to cast the response as a
-converging series featuring orthonormal polynomials.
+The polynomial chaos expansion enables to approximate the output random variable of interest :math:`\vect{Y} = g(\vect{X}) \in \Rset^p` where :math:`\vect{X}` is the input random vector of dimension *n* by the  response surface :
```

Comment:
I wouldn't mention polynomial anywhere in here now that we allow non-polynomial basis.